OasisDEX/oasis-borrow
# Gf/pnl card update

# [Update PnL modal](https://app.shortcut.com/oazo-apps/story/5357/update-pnl-popup-card)


## Changes 👷‍♀️
  - Updated the PnL modal as per designs
  - Mobile view
  - added new values to PnL
  
## How to test 🧪
- head to a multiply vault that has transactions
- go to the overview page and click the Net Value section
- The modal will appear, check that the values are correct

Vault 392 on staging has a vault with history.

Please can we double check my calculations as well to make sure they are as expected.
